### PR TITLE
add apt update to fix some cache issues in Dockerfile

### PR DESCRIPTION
**Pull request by rafmsou**
Hi, 

I was having problems rebuilding the openedx image (`tutor images build openedx`), more specifically those errors:

```
E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.12-1ubuntu0~16.04.12_amd64.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.12-1ubuntu0~16.04.12_amd64.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.12-1ubuntu0~16.04.12_amd64.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.12-1ubuntu0~16.04.12_amd64.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/libx/libx11/libx11-xcb1_1.6.3-1ubuntu2.1_amd64.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl-dev_1.0.2g-1ubuntu4.16_amd64.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl-doc_1.0.2g-1ubuntu4.16_all.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/libx/libx11/libx11-dev_1.6.3-1ubuntu2.1_amd64.deb  404  Not Found [IP: 91.189.88.152 80]

E: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/libx/libx11/libx11-doc_1.6.3-1ubuntu2.1_all.deb  404  Not Found [IP: 91.189.88.152 80]

E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
The command '/bin/sh -c apt install -y libssl-dev zlib1g-dev libbz2-dev     libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev     xz-utils tk-dev libffi-dev liblzma-dev python-openssl git' returned a non-zero code: 100
Error: Command failed with status 100: docker build -t docker.io/overhangio/openedx:10.3.1 /Users/rafmsou/Library/Application Support/tutor/env/build/openedx
```

So I added `apt update` to some `apt install` commands that were failing.

Let me know if anything wrong.
Thanks!

### CODE REVIEW

**Comment by regisb** [comment]
Thanks for you pull request @rafmsou! This looks good to me. Can you please squash your commits and add a message to the changelog before we merge?

**Comment by rafmsou** [comment]
@regisb Done it 🙂

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Note: Breaking changes between versions are indicated by "💥".
 - [Improvement] Make all commands considerably faster
 - [Improvement] Make it easier to override Mongodb connection parameters
 - [Bugfix] Add support for .woff and .woff2 font files in themes (thanks @mrtndwrd!)
+- [Bugfix] Run `apt update` before `apt install` when installing deps in the openedx Dockerfile
```
Comment (regisb) [inline CHANGELOG.md:13]: Sorry to nitpick; can you please move your line in the "Unreleased" section above?

**Comment by rafmsou** [comment]
Yes, of course. My bad, overlooked that.

File: CHANGELOG.md (hunk 1 above)
Comment (rafmsou) [reply CHANGELOG.md]: Done.

**Comment by regisb** [comment]
Thanks for your pr!

**Pull request merged**